resources/extruders/z-bolt_extruder_0.def.json

```diff
@@ -1,14 +1,16 @@
 {
     "version": 2,
-    "name": "Extruder 1",
+    "name": "Toolhead",
     "inherits": "fdmextruder",
-    "metadata": {
+    "metadata":
+    {
+        "machine": "z-bolt_base",
```

Comment:
Did you check if older profiles still work due to this change?

Reply:
Probably no. We want to remove Original and Original Plus profiles, since these printers are no longer officially supported by Z-Bolt, and also printing profiles are outdated. 
New profiles for Limited and Limited Plus are mostly compatible with Original and Original Plus and have been successfully tested by our team.

Reply:
If you want to remove them, you will also need to create a version upgrader script.

Reply:
A version upgrader would make sense if a new profile would be used for the same printer. But in this case it's just that there's a wholly new printer.

Since you say that the profiles for the new printer work on the older printer as well, I think the most logical solution would be to keep the old definition file, but make it inherit from the new printer and use the same quality profile set. That way a version upgrade is only necessary to change the quality profiles, but the definitions can just stay in place.

Reply:
Old profiles are outdated, and adding (also possibly maintaining and updating) new quality profiles and variants will be very difficult. Limited series have same build volume as older Original ones, it's basically just a redesign in all-metal frame with small updates.

Reply:
Yes, so what I meant was to just have one set of profiles, your new profiles, but mark the Original and Original Plus printers to use that same set of profiles (by overriding the `quality_definition` metadata entry).
When those people transition to the new set of profiles, a transformation needs to be made to their configuration. Cura stores which profiles you had active in a "machine instance" resource. That entry would need to change in a version upgrade, to make them use the new set of profiles, rather than the old ones that you don't want to support any more.